ALUM/ALUM/Services/PostSessionService.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  PostSessionService.swift
+//  ALUM
+//
+//  Created by Adhithya Ananthan on 4/26/23.
+//  This file will contain the service function to make a post session route
+//  Note that the function in this file should be added to the SessionService page
+//  once everything is merged
+//
+
+import Foundation
+
+
+struct SessionPostData: Codable {
+    var menteeId: String
+    var mentorId: String
+    var calendlyURI: String
+}
+
+struct PostSessionData: Codable {
+    var sessionId: String
+    var menteeId: String
+    var mentorId: String
+}
+
+class PostSessionService {
```

Comment:
Let's rename this to a SessionService that way it can be added to the session service added by https://github.com/TritonSE/ALUM-Mobile-Application/pull/103/files#diff-2f9a2ce64c523b70bf8358e75454965fbf08561052258bbb2911536e7b14c416